docs/libcurl/symbols-in-versions

```diff
@@ -206,9 +206,11 @@ CURLHEADER_SEPARATE             7.37.0
 CURLHEADER_UNIFIED              7.37.0
 CURLINFO_ACTIVESOCKET           7.45.0
 CURLINFO_APPCONNECT_TIME        7.19.0
+CURLINFO_APPCONNECT_TIME_T      7.60.0
```

Comment:
Since the feature window has closed now for 7.60.0 and this is a change, we can assume this change can land in 7.61.0.

Reply:
Changed... the Travis testing is kicking my ass, though...  Can't get it to pass the tests 1139 and 1140.

Reply:
I don't see it in the most recent commit. I see it fails because of this:
~~~
lib500.c: In function ‘test’:
lib500.c:115:112: error: expected identifier before ‘%’ token
           fprintf(moo, "namelookup vs connect: %" CURL_FORMAT_CURL_OFF_T ".%06ld " %" CURL_FORMAT_CURL_OFF_T ".%06ld\n",
             
~~~

there's a quote in middle of the `".%06ld " %"` which it looks like causes a slew of syntax errors

Reply:
Fixed